resources/views/components/chirp-list.blade.php

```diff
@@ -0,0 +1,46 @@
+<div {{$attributes->merge(['class'=>"mt-6 bg-white shadow-sm rounded-lg divide-y border"]) }}>
+    @foreach ($chirps as $chirp)
+        <div class="p-6 flex space-x-2 border-blue-600">
```

Comment:
add margin between the chirps. look at your chirpers page and note that you have margin between them and they look like cards. do that for the chirps as well so things look consistent

Reply:
also, try adding a little [shadow](https://tailwindcss.com/docs/box-shadow) to your cards